docs/tutorial/tutorial_orbit.rst

```diff
@@ -31,148 +31,182 @@ orbit periods aren't constant, a 100% success rate is not be guaranteed.
 
-If we continue to iterate orbits using
-
-.. code:: python
-
-   ivm.orbits.next()
-
-eventually the next day will be loaded to try and form a complete orbit. You
-can skip the iteration and just go for the last orbit of a day,
+And back an orbit:
 
 .. code:: ipython
 
-   ivm.orbits[-1]
+   f15.orbits.prev()
+   print(f15.index[0], f15.index[-1])
 
-   Returning cnofs ivm  data for 12/29/12
-   Loaded Orbit:1
+   2012-01-01 00:12:21 2012-01-01 01:03:09
 
-.. code:: ipython
+If we continue to iterate orbits using ``f15.orbits.next()`` the next day will
+eventually be loaded to try and form a complete orbit. You can skip the
+iteration and just go for the last orbit of a day using indexing:
 
-   ivm[:5, 'mlt']
+.. code:: ipython
 
-   2012-12-28 23:03:34.160000    0.003109
-   2012-12-28 23:03:35.152000    0.007052
-   2012-12-28 23:03:36.160000    0.010996
-   2012-12-28 23:03:37.152000    0.014940
-   2012-12-28 23:03:38.160000    0.018884
-   Name: mlt, dtype: float32
+   f15.orbits[-1]
+   print(f15.index[0], f15.index[-1])
 
-   ivm[-5:, 'mlt']
+   2012-01-01 23:56:41 2012-01-02 00:47:25
 
-   2012-12-29 00:40:13.119000    23.982937
-   2012-12-29 00:40:14.119000    23.986605
-   2012-12-29 00:40:15.119000    23.990273
-   2012-12-29 00:40:16.119000    23.993940
-   2012-12-29 00:40:17.119000    23.997608
-   Name: mlt, dtype: float32
 
-pysat loads the next day of data to see if the last orbit on 12/28/12 extends
-into 12/29/12, which it does. Note that the last orbit of 12/28/12 is the same
-as the first orbit of 12/29/12. Thus, if we ask for the next orbit,
+pysat loads the next day of data to see if the last orbit on 1 Jan 2012 extends
+into 2 Jan 2012, which it does. Note that the last orbit of 1 Jan 2012 is the
+same as the first orbit of 2 Jan 2012. Thus, if we ask for the next orbit,
 
 .. code:: ipython
 
-   ivm.orbits.next()
+   f15.orbits.next()
+   print(f15.orbits)
+
+   Orbit Settings
+   --------------
+   Orbit Kind: 'polar'
+   Orbit Index: 'gdlat'
+   Orbit Period: Timedelta('0 days 01:37:00')
+   Number of Orbits: 29
+   Loaded Orbit Number: 2
 
-   Loaded Orbit:2
 
 pysat will indicate it is the second orbit of the day. Going back an orbit
-gives us orbit 16, but referenced to a different day. Earlier, the same orbit
-was labeled orbit 1.
+gives us orbit 30, but referenced to a different day. If 2 Jan 2012 had been
+loaded, this would be labeled orbit 1.
 
 .. code:: ipython
 
-   ivm.orbits.prev()
+   f15.orbits.prev()
+   print(f15.orbits)
 
-   Returning cnofs ivm  data for 12/28/12
-   Loaded Orbit:16
+   Orbit Settings
+   --------------
+   Orbit Kind: 'polar'
+   Orbit Index: 'gdlat'
+   Orbit Period: Timedelta('0 days 01:37:00')
+   Number of Orbits: 30
+   Loaded Orbit Number: 30
 
-Orbit iteration is built into ivm.orbits just like iteration by day is built
-into ivm.
+Orbit iteration is built into f15.orbits just like daily iteration is built
+into f15 (see :ref:`tutorial-load`).
```

Comment:
This is currently in the iteration tutorial. We can move it to load if that is better.